aip/general/0162.md

```diff
@@ -342,67 +264,52 @@ ranges of time.
 Revisions are sometimes expensive to store, and there are valid use cases to
 want to remove one or more revisions from a resource's revision history.
 
-APIs **may** define a method to delete revisions, with a structure similar to
-`Delete` ([AIP-135][]) methods:
+APIs **may** define a standard `Delete` ([AIP-135][]) method to delete resource
```

Comment:
this feels a little unnecessary to me - this, along with Get / List could be probably consolidated to a section like "A revision is a collection" or something along those lines, then enumerate that Get / List are required and delete is optional.

Reply:
also, similar to "Getting Revisions" or "Listing Revision", this should be it's own section, correct?